docs/tutorial/new_controller.rst

```diff
@@ -0,0 +1,106 @@
+
+Implementing a Controller
+-------------------------
+
+A controller closes the loop, a class derived from the abstract class
+:py:class:`~pymoskito.simulation_modules.Controller`.
+Its only function calculates the controller output,
+which acts as the actuating variable for the model.
+
+In this scenario, the controller is linear. 
```

Comment:
To keep things simple we start with a controller, based on the linearized system which is given by